index.php

```diff
@@ -0,0 +1,9 @@
+<?php
+require_once __DIR__. '/vendor/autoload.php';
```

Comment:
minor point, it might be worth having a space between __DIR__ and the concatinator '.' 
For example... __DIR__ . '/vendor... etc'

Reply:
Good spot I'll make the change. Thanks